memflow/examples/kernel_structures.rs

```diff
@@ -0,0 +1,126 @@
+/// A simple kernel structuring example using memflow
```

Comment:
Would you mind putting a simple comment on the top of the file with example usage? This usually makes it easier for people to play with it. Something like here: https://github.com/memflow/memflow/blob/main/memflow/examples/cached_view.rs#L1